modules/core/04-channel/keeper/timeout.go

```diff
@@ -158,6 +158,11 @@ func (k Keeper) TimeoutExecuted(
 		k.SetChannel(ctx, packet.GetSourcePort(), packet.GetSourceChannel(), channel)
 	}
 
+	if channel.FlushStatus == types.FLUSHING && !k.hasInflightPackets(ctx, packet.GetSourcePort(), packet.GetSourceChannel()) {
```

Comment:
Just noting that I think the spec needs to be modified here https://github.com/cosmos/ibc/pull/982#discussion_r1238262526

Reply:
keeping an eye on it, will update as required

Reply:
Updated on spec